com.st.STM32CubeMX.metainfo.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.st.STM32CubeMX</id>
+  
+  <name>STM32CubeMX</name>
+  <summary>A graphical tool for configuring STM32 microcontrollers and microprocessors</summary>
+
+  <developer id="com.st">
+    <name>ST Microelectronics</name>
+  </developer>
+  
+  <metadata_license>MIT</metadata_license>
+  <project_license>LicenseRef-proprietary=https://www.st.com/resource/en/license_agreement/dm00218346.pdf</project_license>
+  
+  <description>
+    <p>
+      A graphical tool that allows a very easy configuration of STM32 microcontrollers and microprocessors, as well as the generation of the corresponding initialization C code for the Arm® Cortex®-M core or a partial Linux® Device Tree for Arm® Cortex®-A core), through a step-by-step process.
+    </p>
+  </description>
+
+  <url type="homepage">https://www.st.com/en/development-tools/stm32cubemx.html</url>
+  
+  <launchable type="desktop-id">com.st.STM32CubeMX.desktop</launchable>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://i.ibb.co/P50rgnb/stm32cubemx.png</image>
```

Comment:
if they don't have a screenshot in the website, add the screenshot here. These image host links tend to get lost.

Reply:
They have a [screenshot](https://www.st.com/content/dam/st-crew/developer-zone/cube-mx/stm32cubemx-pinout-configuration%201@3x.png) on their website, but it can't be downloaded with curl

Reply:
ok add it here.